engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/subscription-access-mode.directive.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @ngdoc directive
+ * @name Bastion.subscriptions:subscriptionAccessMode
+ *
+ * @requires contentAccessMode
+ *
+ * @description
+ *   Component for showing information about content access mode (whether content is
+ *   allowed with or without a subscription)
+ */
+angular.module('Bastion.subscriptions').directive('subscriptionAccessMode',
```

Comment:
Since this directive is only ever displayed as a banner when `contentAccessMode === 'org_environment'` I would expect it to be named something more descriptive.  Maybe something like `golden-ticket-banner` or whatever we are calling this mode (even `content-access-mode-banner` would be more descriptive).